src/components/Container/Container.js

```diff
@@ -6,7 +6,7 @@ import './Container.scss';
 const Container = (props) => {
   const tenToDisplay = props.data.slice(0, 10)
   const cardData = tenToDisplay.map((data, index) => {
-  return (<Card {...data} setPath={props.setPath} key={index}/>)
+  return (<Card {...data} setPath={props.setPath} key={index} favorite={props.favorite}/>)
   })
```

Comment:
Nice! This should make maintaining the state of the card a cinch when we go from the movies to characters to favorite back to characters flow